public/pages/Destinations/components/createDestinations/CustomWebhook/QueryParamsEditor.js

```diff
@@ -55,27 +57,31 @@ const handleRenderValueField = (fieldName, index) => (
 const propTypes = {
   type: PropTypes.string.isRequired,
   queryParams: PropTypes.array.isRequired,
+  isEnabled: PropTypes.bool,
 };
 
-const QueryParamsEditor = ({ type, queryParams }) => (
-  <FieldArray
-    name={`${type}.queryParams`}
-    validateOnChange={true}
-    render={arrayHelpers => (
-      <AttributeEditor
-        titleText="Query parameters"
-        onAdd={() => arrayHelpers.push({})}
-        onRemove={index => arrayHelpers.remove(index)}
-        items={queryParams}
-        name={`${type}.queryParams`}
-        addButtonText="Add parameter"
-        removeButtonText="Remove parameter"
-        onRenderKeyField={handleRenderKeyField}
-        onRenderValueField={handleRenderValueField}
-      />
-    )}
-  />
-);
+const QueryParamsEditor = ({ type, queryParams, isEnabled = true }) => {
+  return (
+    <FieldArray
+      name={`${type}.queryParams`}
+      validateOnChange={true}
+      render={arrayHelpers => (
+        <AttributeEditor
+          titleText="Query parameters"
+          onAdd={() => arrayHelpers.push({})}
+          onRemove={index => arrayHelpers.remove(index)}
+          items={queryParams}
+          name={`${type}.queryParams`}
+          addButtonText="Add parameter"
+          removeButtonText="Remove parameter"
+          onRenderKeyField={handleRenderKeyField}
```

Comment:
how about if we could use `isEnabled` directly here instead of passing it to `<AttributeEditor/>`. ? 

```
onRenderKeyField={(fieldName, index) => handleRenderKeyField(fieldName, inedex, isEnabled)}
```
This way we can avoid passing props to component tree and make `<AttributeEditor/>` more generic.

Reply:
I can use the `isEnabled` directly  on `QueryParamsEditor` declaring the functions passed to `onRenderKeyField` and `onRenderValueField` as:
```javascript
const handleKeyField = (fieldName, index) => (handleRenderKeyField(fieldName, index, isEnabled));
const handleValueField = (fieldName, index) => (handleRenderValueField(fieldName, index, isEnabled));
```
replacing 
```javascript
onRenderKeyField={handleRenderKeyField}
onRenderValueField={handleRenderValueField}
```
to
```javascript
onRenderKeyField={handleKeyField}
onRenderValueField={handleValueField}
```
Is that ok or there is a better way to pass `isEnabled` to `handleRenderKeyField` and `handleRenderValueField` without pass to `AttributeEditor`?